tests/core/Observer.spec.ts

```diff
@@ -143,8 +143,8 @@ describe('Observer', async () => {
     // Prepare the mock response from the DownloadManager.
     const didDocumentTemplate = require('../json/didDocumentTemplate.json');
 
-    const [publicKey1, privateKey1] = await Cryptography.generateKeyPairHex('#key1', KeyUsage.recovery);
-    const [publicKey2, privateKey2] = await Cryptography.generateKeyPairHex('#key2', KeyUsage.recovery);
+    const [publicKey1, privateKey1] = await Cryptography.generateKeyPairHex('#extraRecoveryKey1', KeyUsage.recovery);
+    const [publicKey2, privateKey2] = await Cryptography.generateKeyPairHex('#extraRecoveryKey2', KeyUsage.recovery);
```

Comment:
As discussed offline, lets get rid of `generateCreateOperation()` and used the declaritive approach of generating document. ie. `Document.create()`